pkg/dao/templates.go

```diff
@@ -21,7 +22,7 @@ type templateDaoImpl struct {
 	db *gorm.DB
 }
 
-func (t templateDaoImpl) DBToApiError(e error) *ce.DaoError {
+func (t templateDaoImpl) DBToApiError(e error, uuid *string) *ce.DaoError {
```

Comment:
what do you think about either renaming this to TemplateDBToApiError and removing the receiver or also using a receiver on the other DBToApiError funcs and removing the object name? 

or using one DBToApiError method for all objects and just passing in the uuid and the object name? would that work?

Reply:
Some of them check for specific contraints, so as we are doin git today, i'm not sure we can do one method, but yes moving away from a reciever makes sense.